ci/scripts/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
+
+cwd=$(pwd)
+
+pushd $cwd/dp-s3
+  make build
```

Comment:
`Makefile` needs `build` and `audit` targets

Reply:
Yip, I missed them.  Added now thanks.